app/models/name/synonymy.rb

```diff
@@ -13,9 +13,7 @@ def synonym_ids
       if @synonyms
         @synonyms.map(&:id)
       elsif synonym_id
-        Name.connection.select_values(%(
-          SELECT id FROM names WHERE synonym_id = #{synonym_id}
-        )).map(&:to_i)
+        Name.where(synonym_id: synonym_id).pluck(:id)
```

Comment:
Although not related to the subject of this PR, refactored in connection with adding some coverage in order to get Coveralls to pass. See Comment on test/name_test.rb.

This ActiveRecord query is simpler and produces the same SQL as the old version.
`.map(&:to_i)` is unnecessary because `id` is the primary key and ActiveRecord primary keys **are** integers. https://edgeguides.rubyonrails.org/active_record_basics.html#schema-conventions